app/Http/Controllers/ResourceController.php

```diff
@@ -35,11 +35,19 @@ public function index()
      */
     public function create()
     {
+        $resource = Resource::where('user_id', Auth::id())->first();
+
         $user = [
```

Comment:
okay in that case can we change this to `resourceDetails`? 

because `user` model can be `hirer` or `resource` and i want to make sure we provide some context that we are dealing with a resource here (_but not a hirer_).

Reply:
ok, we will do this.